### ISSUE TITLE
impl std::io::Write for TinyVec<[u8; x]>

### ISSUE THREAD

**Issue by eeeebbbbrrrr** [Lokathor/tinyvec#151]
Would y'all accept a PR that implements `std::io::Write` for u8-based TinyVecs?

If so, I'd be thrilled to submit it.

**Comment by Lokathor** [comment]
Yes. I'd love it. It should be behind a feature named `std` but otherwise I think you can figure out the rest.

**Comment by eeeebbbbrrrr** [comment]
Consider it done.  Thanks!

**Comment by eeeebbbbrrrr** [comment]
I have a question regarding a `std` feature...

How do I use that to defeat the ``#![no_std]` at the top of `src/lib.rs`?  This is probably my lack of Rust knowledge showing...

**Comment by Lokathor** [comment]
you'd change that to `#![cfg_attr(not(feature = "std"), no_std)]`

**Comment by eeeebbbbrrrr** [comment]
Thank you.  I'd have never figured that out.

I'll put up a PR tomorrow.  It was overly simple to implement.  Y'all did a great job with TinyVec.  Every time I find a use for it in my projects performance improves measurably.  It's such a valuable crate.

**Comment by fidelicura** [comment]
Hi! Shouldn't this be closed? #152 is merged, which fixes this issue.
Anyway, what about `core::fmt::Write`?
